### PR TITLE
clear cvar when re-calling inject_static

### PR DESCRIPTION
**Pull request by nerdrew**
- version bump
- test cleanup
- don't include #inject and #inject_static as instance methods

### CODE REVIEW

File: spec/interjectable_spec.rb [hunk 1]
```diff
@@ -96,7 +103,7 @@
   end
 
   context "when included" do
-    let(:klass) { Class.new { extend Interjectable } }
+    let(:klass) { Class.new { include Interjectable } }
```
Comment (zachmargolis) [inline spec/interjectable_spec.rb]: LOL great catch

Comment (zachmargolis) [reply spec/interjectable_spec.rb]: Can we also add an assertion of like `klass.instance_method(:inject)` is nil?

Comment (nerdrew) [reply spec/interjectable_spec.rb]: I added tests that try calling the non-existent methods.

Comment (nerdrew) [reply spec/interjectable_spec.rb]: Mind if I merge?

Comment (zachmargolis) [reply spec/interjectable_spec.rb]: 🚢 it! 👍

**Pull request merged**